tools/deployment/package-helm/templates/aws-config-secret.yaml

```
@@ -0,0 +1,15 @@
+{{- if .Values.clpConfig.aws_config }}
+apiVersion: "v1"
+kind: "Secret"
+metadata:
+  name: {{ include "clp.fullname" . }}-aws-config
+  labels:
+    {{- include "clp.labels" . | nindent 4 }}
+stringData:
+  {{- with .Values.clpConfig.aws_config.credentials }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)